README.md

```diff
@@ -147,6 +148,8 @@ Create a verifier function by calling `createVerifier` and providing one or more
 
 - `cacheTTL`: The maximum time to live of a cache entry (in milliseconds). If the token has a earlier expiration or the verifier has a shorter `maxAge`, the earlier takes precedence. The default is `600000`, which is 10 minutes.
 
+- `errorCacheTTL`: A number or function `function (tokenError) => number` that represents the maximum time to live of a cache **error** entry (in milliseconds).Example: the `key` function fails or does not return a secret or public key. The default is `0`.
```

Comment:
explain what the default value of 0 means. it means that errors are never cached, right? also note (correct me if I'm wrong) that this behavior is a breaking change (because errors before were cached in the same way as non errors, right?), so we'll have to release a new semver major

Reply:
yes this is a breaking change since now errors are not cache by default